pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -143,15 +147,17 @@ private static void initialize() {
         }
     }
 
-    public static synchronized void initClient() {
+    public static void initClient() {
```

Comment:
Safe to remove synchronization because this is only called from already-synchronized 'initialize()'